ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePage.tsx

```diff
@@ -63,8 +65,17 @@ export const imageDetailsQuery = gql`
     }
 `;
 
-function ScannerV4RequiredTooltip({ children }: { children: ReactElement }) {
-    return <Tooltip content="SBOM generation requires Scanner V4">{children}</Tooltip>;
+function GenerateSbomButtonTooltip({
```

Comment:
What do you think about **Optional**SbomButtonTooltip?

Reply:
I like it. On the topic of nouns that read like commands, what do you think of `SbomGenerationModal` to replace `GenerateSbomModal`?

Reply:
Neutral.
* In general, the less transformations, the more I succeed with **Find in Files**
* In this specific case, `GenerateSbom` is already transformed for camel case, so go for the change if you like.